content/developer/vanilla-cli/index.md

```diff
@@ -140,3 +141,39 @@ Convert addons from using `$PluginInfo` / `$ThemeInfo` / `about.php` metadata de
 #### `--vanillasrc`
 
 You vanilla installation's source directory.
+
+## Common Issues
+
+There are a few common issues that people run into.
+
+### I'm getting an error message that my Vanilla source directory is missing or incorrect.
+
+In order to function properly the CLI tool needs to know where your Vanilla installation is located on your system. If my vanilla installation is at `~/workspace/vanilla` then I can either pass it as a parameter on every command:
+
+```bash
+vanilla build --vanillasrc=~/workspace/vanilla
+```
+
+or set the environmental variable.
```

Comment:
I know it seems obvious, but mention that they need to replace the path for their own. Try and write these docs from the perspective of them being used by a web integrator that doesn't necessarily have a lot of experience with the terminal.